src/corner/core.py

```diff
@@ -739,9 +759,15 @@ def overplot_points(fig, xs, **kwargs):
     xs = _parse_input(xs)
     K = len(xs)
     axes = _get_fig_axes(fig, K)
-    for k1 in range(K):
-        for k2 in range(k1 + 1, K):
-            axes[k2, k1].plot(xs[k1], xs[k2], **kwargs)
+    if reverse:
```

Comment:
I think there's an indexing issue here. We don't want the points to appear on the diagonal (on the histogram plots).

Reply:
Again, my bad. This should be up to `k1` on line 764. I'll push again.